Sources/PromotedAIMetricsSDK/ClientConfig/ClientConfig.swift

```diff
@@ -11,8 +12,10 @@ public class ClientConfig: NSObject {
   var loggingEnabled: Bool = true
   
   @objc public var metricsLoggingURL: String = ""
+  @objc public var devMetricsLoggingURL: String = ""
```

Comment:
For web, we'll probably have different environment params for each of these that fill in the same values for dev vs prod depending on the case.  So dev would reuse `metricsLoggingURL` but have different values.  I can see this changing then when we're further along and use a dynamic config solution.

Reply:
We can replicate this behavior by duplicating the URL but with different params. Let me know how this shapes up on the server side and we can update this.